velox/connectors/tpch/tests/TpchConnectorTest.cpp

```diff
@@ -170,6 +171,45 @@ TEST_F(TpchConnectorTest, unknownColumn) {
       VeloxUserError);
 }
 
+// Join nation and region.
+TEST_F(TpchConnectorTest, join) {
+  auto planNodeIdGenerator =
+      std::make_shared<exec::test::PlanNodeIdGenerator>();
+  core::PlanNodeId nationScanId;
+  core::PlanNodeId regionScanId;
+  auto plan =
+      PlanBuilder(planNodeIdGenerator)
+          .tableScan(
+              tpch::Table::TBL_NATION, {"n_regionkey"}, 1 /*scaleFactor*/)
+          .capturePlanNodeId(nationScanId)
+          .hashJoin(
+              {"n_regionkey"},
+              {"r_regionkey"},
+              PlanBuilder(planNodeIdGenerator)
+                  .tableScan(
+                      tpch::Table::TBL_REGION,
+                      {"r_regionkey", "r_name"},
+                      1 /*scaleFactor*/)
+                  .capturePlanNodeId(regionScanId)
+                  .planNode(),
+              "", // extra filter
+              {"r_name"})
+          .singleAggregation({"r_name"}, {"count(1) as nation_cnt"})
+          .orderBy({"r_name"}, false)
+          .planNode();
+
+  auto output = getResults(
```

Comment:
should we also check from the execution stats that the number of dynamics filters is 0?

Reply:
We could, but TPC-H doesn't support dynamic filters. If these were pushed down we would get an error (that's what happens now without the change).

Reply:
Got it. So the lack of error means that dynamic filters are not being pushed down.

Reply:
That's right.